engine-precompiles/Cargo.toml

```diff
@@ -17,7 +17,7 @@ aurora-engine-types = { path = "../engine-types", default-features = false }
 aurora-engine-sdk = { path = "../engine-sdk", default-features = false }
 base64 = { version = "0.13.0", default-features = false, features = ["alloc"] }
 borsh = { version = "0.8.2", default-features = false }
-bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false }
+bn = { package = "aurora-bn", git = "https://github.com/birchmd/aurora-bn.git", rev = "0ae7099d379e53d36f1883cf0d840edf67f71e7d", default-features = false }
```

Comment:
We would not need to use my fork if https://github.com/aurora-is-near/aurora-bn/pull/1 was reviewed and merged cc @joshuajbouw

Reply:
We don't need to use aurora-bn anymore at all with the new changes. Now uses zeropool-bn which has everything that we wanted from it :).